package/environments/types.js

```
@@ -0,0 +1,6 @@
+"use strict";
```

ESLint

[error] 'use strict' is unnecessary inside of modules.

(strict)